footer.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">    
+        <link rel="stylesheet-footer" href="CSS/footer.css">
+    </head>
+
+    <body>
+      <div class="mobile-footer">
+
+        <div class="h1-footer">
+        <h1>Cakes</h1>
+        </div>    
+
+        <div class="nav-footer">
+
+          <div class="links-footer">
+            <a href="#our-cakes">Наши торты</a>
+            <a href="#personal-order">Индивидуальный заказ</a>
+            <a href="#reviews">Отзывы</a>
+            <a href="#about-us">О нас</a>
+            <a href="#our-advantages">Наши преимущества</a>
+            <a href="#contacts">Контакты</a>
+          </div>
```

Comment:
This block should be wrapped in `<nav>` navigation tag. Also eac anchor tag `<a>` should be wrapped in list item `<li>`.